.github/actions/create-local-test-infra-resources/action.yaml

```diff
@@ -44,6 +44,10 @@ runs:
       run: oc wait pod --for=condition=ready -l "app=${{ inputs.tpp_pod_label_app }}" --timeout=${{ inputs.oc_pod_timeout }}
       shell: bash
 
+    - name: Display all pods in namespace
+      run: oc get pods -o wide
```

Comment:
it need -A or  -n namespace, otherwise no output

Reply:
The namespace is set in the lines above